### DISCUSSION TITLE
How to Uninstall Streamlink (v2.0.0) in Windows 10?

### DISCUSSION THREAD

**Discussion by BoopFake7** [streamlink/streamlink#3690] [Q&A]
Hello, last month I installed 'streamlink-2.0.0.exe' progarm as well with 'streamlink-twitch-gui-v1.11.0-win64-installer.exe' to complete and launched Streamlink Twitch GUI app in my Windows 10 PC.
But now I want to uninstall both Streamlink and Streamlink Twitch GUI apps.

So far I have already uninstall Streamlink Twitch GUI, by going through it's "C:\Program Files" folder and click "Uninstall Streamlink Twitch GUI.exe" to remove it.

So my main question is, How do I properly uninstall Streamlink app?

Do I need to going through it's C:\Program Files folder to find it? or just go to Windows 10's Control Panel - Progarms and Features to uninstall it?

**Comment by bastimeyer** [comment]
> Do I need to going through it's C:\Program Files folder to find it? or just go to Windows 10's Control Panel - Progarms and Features to uninstall it?

Both methods will execute the uninstaller. There is no difference.

**Reply by BoopFake7** [reply]
Hi bastimeyer, thanks for answering my question.

Just now I have already uninstalled Streamlink app by using Control Panel - Progarms and Features, but for some reason there is another file/app (or what left) that took it's place. But this time the Streamlink Blue chain logo is now an '.exe' image file?

So my question is, Do I need to uninstall the 'second' Streamlink app again?

(Also I couldn't find the Streamlink app within both 'C:\Program Files' and 'C:\Program Files (x86)' folders)
![Streamlink second hidden file](https://user-images.githubusercontent.com/82467938/114679334-f8d36b80-9d03-11eb-8a39-ec003a0e9120.PNG)

**Reply by parvinders347** [reply]
same issue with me
